posthog/batch_exports/debug/debugger.py

```diff
@@ -346,7 +347,19 @@ def iter_run_record_batches_from_s3(
             base_dir=f"{settings.BATCH_EXPORT_INTERNAL_STAGING_BUCKET}/{folder}", recursive=True
         )
 
-        for file_info in self.s3fs.get_file_info(file_selector):
+        # get the most recent attempt
```

Comment:
Maybe we should let the developer pass an attempt? Just in case we want to check earlier attempt data (For whatever reason). Latest as the default sounds fine though.